sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/ServerAccessDataService.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2020 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.symeda.sormas.backend.sormastosormas;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import javax.ejb.LocalBean;
+import javax.ejb.Stateless;
+import javax.inject.Inject;
+
+import com.opencsv.CSVReader;
+
+import de.symeda.sormas.api.SormasToSormasConfig;
+import de.symeda.sormas.api.utils.CSVUtils;
+
+@Stateless
+@LocalBean
+public class ServerAccessDataService {
+
+	private static final String SERVER_ACCESS_DATA_FILE_NAME = "server-access-data.csv";
+
+	private static final String SERVER_LIST_FILE_NAME = "server-list.csv";
+
+	@Inject
+	private SormasToSormasConfig sormasToSormasConfig;
+
+	public Optional<ServerAccessData> getServerAccessData() {
+		Path inputFile = Paths.get(sormasToSormasConfig.getFilePath(), SERVER_ACCESS_DATA_FILE_NAME);
+		try (Reader reader = Files.newBufferedReader(inputFile, StandardCharsets.UTF_8);
+			CSVReader csvReader = CSVUtils.createCSVReader(reader, ',')) {
+			return Optional.of(buildServerAccessData(csvReader.readNext()));
+
+		} catch (IOException e) {
+			return Optional.empty();
+		}
+	}
+
+	public List<ServerAccessListItem> getServerList() {
+		Path inputFile = Paths.get(sormasToSormasConfig.getFilePath(), SERVER_LIST_FILE_NAME);
+		try (Reader reader = Files.newBufferedReader(inputFile, StandardCharsets.UTF_8);
+			CSVReader csvReader = CSVUtils.createCSVReader(reader, ',')) {
+			return csvReader.readAll()
+				.stream()
+				// skip the empty line at the end
```

Comment:
actually skips all lines without commas

Reply:
yes but normally csvs don't have empty lines in the middle

Reply:
If only one column was present, this would ignore the complete file.
I don't think this is a problem here, this way adding comments is already supported 😃 
To me this is a config file with entries in csv format. So empty lines inbetween and comments can really make sense here.

Reply:
I see your point, but to me this is a generated file and manipulated by scripts and I wouldn't complicate with sophisticated validations and other stuff because this is a temporary solution until we implement the 'central server'.

Reply:
This was actually only about the comment that felt misleading to me.
I actually first thought about using csv for values in a properties file in order to not have multiple lines per server.
Then I thought that it would easier no not mix the formats.
We have provided a script for adding entries, but not for removing or modifying. So manual editing will be required.